client-ui/src/index.html

```diff
@@ -30,7 +30,8 @@
   // Uncomment it to set Spline client configuration properties:
 
   window['SplineConfiguration'] = {
-    apiUrl: "PUT_YOUR_SPLINE_REST_ENDPOINT_URL_HERE"
+    apiUrl: "PUT_YOUR_SPLINE_REST_ENDPOINT_URL_HERE",
+    embeddedMode : true => hide header / false => display full app with header
```

Comment:
Won't it break our Client Web app?
See `WebController.index(...)`  method. It replaces the placeholder and uncomments this block.

Reply:
Well spotted. Thanks